modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -170,15 +171,35 @@ Because the following endpoints use mutual TLS authentication to establish the c
 .2+| 443 .2+| HTTPS | US control plane: `ecr.us-east-1.amazonaws.com` | Runtime Fabric Docker repository.
 | EU control plane: `ecr.eu-central-1.amazonaws.com` | Runtime Fabric Docker repository.
 .2+| 443 .2+| HTTPS | US control plane: `494141260463.dkr.ecr.us-east-1.amazonaws.com` | Runtime Fabric Docker repository.
-| EU control plane: `494141260463.dkr.ecr.eu-central-1.amazonaws.com` | Runtime Fabric Docker repository.
+| EU control plane: `655988475869.dkr.ecr.eu-central-1.amazonaws.com` | Runtime Fabric Docker repository.
 .2+| 443 .2+| HTTPS | US control plane: `api.ecr.us-east-1.amazonaws.com` | Runtime Fabric Docker repository.
 | EU control plane: `api.ecr.eu-central-1.amazonaws.com` | Runtime Fabric Docker repository.
 .2+| 443 .2+| HTTPS | US control plane: `prod-us-east-1-starport-layer-bucket.s3.amazonaws.com` `prod-us-east-1-starport-layer-bucket.s3.us-east-1.amazonaws.com` | Runtime Fabric Docker image delivery.
 | EU control plane: `prod-eu-central-1-starport-layer-bucket.s3.amazonaws.com` `prod-eu-central-1-starport-layer-bucket.s3.eu-central-1.amazonaws.com` | Runtime Fabric Docker image delivery.
 .2+| 443 .2+| HTTPS | US control plane: `runtime-fabric.s3.amazonaws.com` | Runtime Fabric Docker repository.
 | EU control plane: `runtime-fabric-eu.s3.amazonaws.com` | Runtime Fabric Docker repository.
+.2+| 443 .2+| HTTPS | US control plane: `configuration-resolver.prod.cloudhub.io` | Anypoint Configuration Resolver.
+| EU control plane: `configuration-resolver.prod-eu.msap.io` | Anypoint Configuration Resolver.
 |===
 
+=== Mutual Authentication with Anypoint
+
+For Runtime Fabrics running version 1.4.x and above, the user have the option to enable mutual authentication with 
+Anypoint connectivity for the cluster from the  settings page in Runtime Manager. Enabling mutual authentication will
+provide better security for communications between the Runtime Fabric cluster and Anypoint platform, and is highly recommended.
+
+Before you enabled multual authentication, please make sure the following endpoint and port is whitelisted and reachable from the cluster:
+
+* `configuration-resolver.prod.cloudhub.io:443`
```

Comment:
What about the eu endpoint? Can we just reference them to the `hostname to whitelist`section?

Reply:
This section seems odd to put in the "Prerequesites" page. That page is designed for a new customer who wants to install RTF.

I'd consider moving this to its own page under the "Manage Runtime Fabrics" section, or possibly add to the "Limitations" section.

Reply:
I created a separate topic - see if this works better. (I didn't remove it from the other topic yet in case you decide not to separate it out.)